### ISSUE TITLE
Configure continuous integration.

### ISSUE THREAD

**Issue by JurajBurian** [JurajBurian/mailer#5]
Configure continuous integration.
see: https://github.com/marketplace/category/continuous-integration

**Comment by basert** [comment]
I guess Travis CI is the most popular one and it's free for OSS projects. It's also pretty easy to setup.

**Comment by JurajBurian** [comment]
Yes Travis looks as good choice. We can do it :-), and also  we can put fancy icons about build statuses and so on ....
